LayoutTests/TestExpectations

```diff
@@ -3795,6 +3795,7 @@ webgl/2.0.y [ Skip ]
 # Explicitly enable tests which we have fixed and do not have corresponding 2.0.0 test functionality.
 webgl/2.0.y/conformance/canvas/to-data-url-test.html [ Pass ]
 webgl/2.0.y/conformance/misc/invalid-passed-params.html [ Pass ]
+webgl/2.0.y/conformance/misc/expando-loss.html [ Pass ]
```

Comment:
Does this test cover what is fixed here? Given the name of the test that seems surprising.

Reply:
This test iterates over different attachment points while re-binding renderbuffers and requesting their names (JS objects). Without this PR, calling `getFramebufferAttachmentParameter` with `DEPTH_STENCIL_ATTACHMENT` fails in WebGL 2.0 because the semantic of this attachment point is different than in WebGL 1.0.